### ISSUE TITLE
404 webserver but says its running

### ISSUE THREAD

**Issue by 91ajames** [pi-hole/pi-hole#2741]
- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).

**How familiar are you with the the source code relevant to this issue?:**
7

---
**Expected behaviour:**
load the page

**Actual behaviour:**
404 nginx error
nginx/1.10.3

**Debug token provided by [uploading `pihole -d` log](https://tricorder.pi-hole.net/741b6679ho!):**

**Troubleshooting undertaken, and/or other relevant information:**

`{Steps of what you have done to fix this}`

Happen about a week ago but fixed it due to a apt-get update cache error from '/etc/apt/sources.list.d' update list. Happened again and done the same thing and update pihole, success but now can't load the webpage this time like I did last time.

I know I'm having an issue updating php7.0 to 7.1 which i still need for another webserver thats on a different port. One of the reasons why I was having issues updating and probably has affected pihole as well. After I fixed it the 1st time, my graph bar was reset'd and the numbers were reset but kept all my whitelist and blacklist urls intact.

Asus Tinkerboard - Tinker-OS

edit:
lighttpd failing to start.

lighttpd config

- [ ] 
[lightp.txt](https://github.com/pi-hole/pi-hole/files/3158056/lightp.txt)

Similar to this situation but haven't had much success.
https://discourse.pi-hole.net/t/admin-interface-not-reachable/1242

**Comment by dschaper** [comment]
`nginx` is not part of the supported configuration. We use `lighttpd` and port 80, if you have another web server on that same port (`nginx`) there will be conflicts. I would be able to check that but there is no token provided as was requested in the template.

**Comment by 91ajames** [comment]
its there, i just got worded with "uploading pihole -d log:" https://tricorder.pi-hole.net/741b6679ho! But even I can't access it.

**Comment by dschaper** [comment]
You won't be able to access it, the link is only for the developers. You can see a copy of what was sent in your `/var/log` directory. The sanitized Pi-hole debug log is the file that is uploaded.

**Comment by dschaper** [comment]
You have configured `ngnix` to respond where `lighttpd` is required. See the below snippets showing `nginx` answering for our debug queries.

```
*** [ DIAGNOSING ]: Dashboard and block page
[âœ—] Block page X-Header: X-Header does not match or could not be retrieved.
HTTP/1.1 200 OK
Server: nginx/1.10.3
Date: Wed, 08 May 2019 14:10:44 GMT
Content-Type: text/html; charset=UTF-8
Connection: keep-alive

[âœ—] Web interface X-Header: X-Header does not match or could not be retrieved.
HTTP/1.1 404 Not Found
Server: nginx/1.10.3
Date: Wed, 08 May 2019 14:10:44 GMT
Content-Type: text/html
Content-Length: 169
Connection: keep-alive
```

**Comment by 91ajames** [comment]
[quote]You won't be able to access it[/quote]
I thought so.

So after I ran "systemctl stop nginx" and restarted lighttpd, I was able to access pihole again. 
But I've had nginx running and lighttpd running at the same time so what caused it to stop all of a sudden? Weird bc I hadn't changed anything in a while.

The other web was Organizr on a different port which i haven't used in a while bc of the php problem.
